docs/rules/newline-per-chained-call.md

```diff
@@ -23,12 +23,29 @@ d3
         42
     ])
     .enter()
-    .append('p')
+    .append("p")
     .text(function (d) {
         return "I'm number " + d + "!";
     });
 ```
 
+Another argument in favor of this style is that it improves the readability of diffs when something in the method chain is changed:
+
+```diff
+// More readable
+d3
+    .select("body")
+    .selectAll("p")
+-    .style("color", "white");
++    .style("color", "blue");
+```
+
+```diff
+// Less readable
+-d3.select("body").selectAll("p").style("color", "white");
++d3.select("body").selectAll("p").style("color", "blue");
+```
+
 This rule reports such code and encourages new lines after each call in the chain as a good practice.
```

Comment:
Somewhat beyond the issue, recommend deleting this sentence at the end of the introduction to avoid ambiguity which code is “such code” and because the purpose of the first sentence under Rule Details is to make this point.